src/util/amount.rs

```diff
@@ -135,14 +135,14 @@ fn parse_signed_to_satoshi(
     mut s: &str,
     denom: Denomination,
 ) -> Result<(bool, u64), ParseAmountError> {
-    if s.len() == 0 {
+    if s.is_empty() {
         return Err(ParseAmountError::InvalidFormat);
     }
     if s.len() > 50 {
         return Err(ParseAmountError::InputTooLarge);
     }
 
-    let negative = s.chars().next().unwrap() == '-';
+    let negative = s.starts_with('-');
```

Comment:
It's not problem with this patch, but since I'm looking at it: I'd name this variable `is_negative` so it's clear its a boolean.

Reply:
Done in #309